usr/src/lib/libzpool/Makefile.com

```diff
@@ -61,6 +61,7 @@ CLEANFILES += $(EXTPICS)
 
 $(LINTLIB) := SRCS=	$(SRCDIR)/$(LINTSRC)
 $(LINTLIB): ../common/zfs.h
```

Comment:
i try upstream my changes what i did on dilos.
if it is not applicable - just drop it.
i try to provide universal solution.